js/components/EmailInviteIndicator/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { PropTypes } from 'react';
+import { View, Text } from 'react-native';
+import styles from './styles';
+
+const EmailInviteIndicator = ({ number }) => {
+  return (
+    <View>
+      <Text>{number || 0}<Text> Invites</Text></Text>
```

Comment:
Yes. The UI needs these text elements to be differently styled.  I added the structure for later style props.